meilisearch-http/src/helpers/tide.rs

```diff
@@ -55,7 +55,7 @@ impl RequestExt for Request<Data> {
     fn url_param(&self, name: &str) -> SResult<String> {
         let param = self
             .param::<String>(name)
-            .map_err(|_| ResponseError::bad_parameter("identifier", name))?;
+            .map_err(|e| ResponseError::bad_parameter(name, e))?;
```

Comment:
Why do you change that? Does this mean the identifier is now the name of it directly (like "objectId" or "documentId")?

Reply:
No, it's just because previously the result wasn't expected. 

The error should be `Bad parameter {name}; {err}` and not `Bad parameter identifier; {name}`